spec/03-plugins/09-key-auth/02-access_spec.lua

```diff
@@ -569,19 +569,16 @@ for _, strategy in helpers.each_strategy() do
         end)
       end
 
-      it("fails with 'key_in_body' and unsupported content type", function()
+      it("works with 'key_in_body' and unsupported content type", function()
         local res = assert(proxy_client:send {
-          path = "/status/200",
+          method  = "GET",
+          path    = "/request?apikey=kong",
           headers = {
             ["Host"] = "key-auth6.com",
             ["Content-Type"] = "text/plain",
-          },
-          body = "foobar",
+          }
         })
-
-        local body = assert.res_status(400, res)
-        local json = cjson.decode(body)
-        assert.same({ message = "Cannot process request body" }, json)
```

Comment:
What happens in this test case that was removed? That should still produce an HTTP 400, right? (Even if with a different error message)

Could you keep this test case in the test suite and then add the new one, instead of replacing one with the other? Thank you!

Reply:
OK.